cpp/data/rss_email.template.en

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+Dear {user_name},
+
+The following items from your selected RSS feeds are new:</br>
+    <table>
+        <thead>
+            <tr>
+                <th>Item Title</th>
+                <th>Feed</th>
+                <th>Item URL</th>
+            </tr>
+        </thead>
+        <tbody>
+        {LOOP item_titles,feed_url,feed_name,item_urls}
```

Comment:
Loop variables should all be in plural form here as well as inside the loop

Reply:
Done